ArchiSteamFarm/Steam/Integration/ArchiHandler.cs

```diff
@@ -205,7 +205,8 @@ public async IAsyncEnumerable<Asset> GetMyInventoryAsync(uint appID = Asset.Stea
 
 			get_descriptions = true,
 			steamid = steamID,
-			count = itemsCountPerRequest
+			count = itemsCountPerRequest,
+			language = language ?? ""
```

Comment:
I don't think we should be setting the property at all if it's `null`, let alone duplicate the (current) default value from an external library. In my opinion this should be an `if (language != null) request.language = language;` below the object initializer.